notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -208,6 +217,7 @@ public NoteServiceImpl(DataStorage dataStorage,
     this.cmsService = cmsService;
     this.fileService = fileService;
     this.uploadService = uploadService;
+    this.metadataService = metadataService;
```

Comment:
we have also ImageThumbnailService thumbnailService = CommonsUtils.getService(ImageThumbnailService.class); used in getNoteFeaturedImageInfo to be modified also

Reply:
Done